std_msgs/Header header
string[] name
float64[] q
float64[] dq
float64[] qdot
float64[] dqdot
float64[] ctrl_input
